otj-metrics-actuate/src/main/java/com/opentable/metrics/actuate/health/HealthIndicatorCodahaleConfiguration.java

```diff
@@ -16,7 +16,9 @@
 
 /**
  * Registers {@link HealthIndicator} beans in the {@link HealthCheckRegistry}.
- *
+ * In other words takes Spring Actuator HealthIndicators and converts to DropWizard Healthchecks
+ * Dmitry: Is there any weirdisms about effectively having duplicate Health checks (eg both DropWizard and Actuator?)
+ * Couldn't that cause an infinite loop in addition to duplicate checks?
```

Comment:
@mikebell90 The trick is here:
https://github.com/opentable/otj-metrics/blob/3ae8cf625be28025999214c2313a934955f3a710/otj-metrics-actuate/src/main/java/com/opentable/metrics/actuate/health/HealthIndicatorCodahaleConfiguration.java#L40